File: sample.cpp
#include <iostream>
#include <cmath>
using namespace std;
//function for asking repeat the program
bool askRepeat(){
    cout<<"What would you like to do next?\n[1] Use another Operation\n[2] No more\n";
    bool wrong=true;
    int a;
    do{
    cout<< "Command : ";
    cin>>a;
    if (a==1)
        return true;
    else if (a==2){
        cout<<"You are now exiting the program.\n Thank you!";
        return false;
        }
    else
        cout<<"Command Invalid!\n";
    }while(wrong);

}
//functions used for Basic Calculator
void raise(float x,float y,float& z){
    z=x;
    while(y!=1){
        z*=x;
        y--;
    }

 } 
int add(float a,float b){
    return a+b;
}
int sub(float a,float b){
    return a-b;
}
int mul(float a,float b){
    return a*b;
}
int div(float a,float b){
    return a/b;
} 
void basicCalculator(){
float a,b,c=0;
     char d;
        cout<<"Please enter the first number\n";
        cin>>a;
        cout<<"\nPlease enter the second number\n";
        cin>>b;
        for (int i=0;i<1;i++){
            cout<<"\nPlease choose what operation to use: + - x / ^ : ";
            cin>>d;
        switch(d){
        case '+':{  
            c=add(a,b);   
            break;
        }
        case '-':{
            c=min(a,b);
            break;
        }  
        case 'x':{
            c=mul(a,b);
            break;
        }
        case '/':{
            div(a,b);
            break;        
        }
        case '^':{
            raise(a,b,c);
            break;
        }
        default:{
            cout<<"\nPlease Enter a Valid Operation!\n";
            i--;
            
        }
               
        }//end of switch
        }//end of loop
        switch(d){
        case '+':{  
            cout<<"The sum of the two inputs is: "<<c<<endl;
            break;
        }
        case '-':{
            cout<<"The difference of the two inputs is: "<<c<<endl;
            break;
        }  
        case 'x':{
            cout<<"The product of the two inputs is: "<<c<<endl;
            break;
        }
        case '/':{
            cout<<"The quotient of the two inputs is: "<<c<<endl;
            break;        
        }
        case '^':{
            cout<<"The raised value of "<<a<<" is: "<<c<<endl;
            break;
        }

        }
}


 int main(){
     int op;
     bool repeat =true;
     do{
     cout<<"\nWhat Operation would you like to use?\n[1] Basic Calculator\n[2] Create Array \n[0] Exit the Program\n\nCommand : ";
     cin>>op;
     switch (op){
         case 0:{
             cout<<"You are now exiting the program.\n Thank you!";
             repeat=false;
             break;
         }
         case 1:{
             cout<<"Greetings! you are now using Basic Calculator\n";
             basicCalculator();
             repeat= askRepeat();
             break;
         }
         case 2:{
             cout<<"Greetings! You are now using Array Creation\n";
             repeat = askRepeat();
             break;
         }
         default:{
             cout<<"Error try again\n";
         }
     }
     }while(repeat);
return 0;
 }

File: sample.py
from xmlrpc.client import boolean
#function for Basic Calc
#recursion for getting the factorial
def getFactorial(num1):
    if num1 ==1:
        return 1
    else:
        num2 = num1 -1 
        return(num1 *getFactorial(num2))
next_num = 1
prev_num = 0
total_num = 0
def Fibonacci(num1):
    global next_num
    global prev_num
    global  total_num
    if(num1 >0):
        total_num = prev_num + next_num
        prev_num = next_num
        next_num = total_num
        print(total_num," ")
        Fibonacci(num1-1, end="")#Use end="" to prevent compiler to create newline

def Basic_Calc():
    num1 = int(input("Enter your 1st number: "))
    num2 = int(input("Enter your 2nd number: "))
    op = int(input("What do you want to do?\n[1]ADD\t\t[2]SUBTRACT\n[3]TIMES\t[4]DIV\n[5]REMAINDER\t[6]FACTORIAL\n[7]Fibonacci\nDecision : "))
    if op == 1:
        print (num1+num2)
    elif op ==2:
        print(num1-num2)
    elif op ==3:
        print(num1*num2)
    elif op ==4:
        print(num1/num2)
    elif op == 5:
        print(num1%num2)
    elif op == 6:
        num1 = int(input("Enter a base value to factorial: "))
        print("The factorial of",num1,"is",getFactorial(num1))
    elif op == 7:
        num1 = int(input("How many # to display on fibonacci series: "))
        Fibonacci(num1)
    else:
        print("Invalid input")
#function for MAD LIB
def Mad_Lib():
    color = input("Enter a color:")
    pnoun = input("Enter a plural noun:")
    celeb = input("Enter a celebrity: ")
    print ("Roses are",color.lower())
    print(pnoun.lower().title(),"are blue\nI love ", celeb.lower().title())

#START
repeat = True
while repeat:
    app = int(input("What application would you like to use?\n[1]Basic Calculator\n[2]Mad Libs\n[3]EXIT\nDecision : "))
    if app == 1:
        Basic_Calc()
    elif app == 2:
        Mad_Lib()
    elif app == 3:
        print("You are now exiting the Program!\nThanks for trying it")
        repeat = False
    else:
        print("Wrong Input! Try Again")
    print("=============================")